content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,89 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
```

Comment:
Where is istiod for this change notes?   That is the most important thing for 1.5

Reply:
@howardjohn is on it